src/services/twilio-api/get-help-doc-link.js

```diff
@@ -0,0 +1,60 @@
+const jsonMap = require('../api-doc-mapping.json');
+// url: {path : {topics: {}, commands: {}}, subpath : {wherever applicable}}
+const urlMap = new Map(Object.entries(jsonMap));
+
+const getSubroute = (cmdDetailsArr, subpathMap) => {
+  const subdomain = cmdDetailsArr[3];
```

Comment:
Does hardcoding the index work for every command? 
'api:conversations:v1:services:conversations:participants:update' -> would give us services but,
'api:core:accounts:create'  -> would give us crete

Reply:
Good point. The url generation has two parts to it, root path generation and subroute generation. In such case, 'api:core:accounts:create', the root path would be generated on the basis of "api:core", the subroute however needs a version to find the relevant document link (why, because it was a common bifurcation pattern in API docs). "api:core" is one of the outliers because it doesn't have versioning in possible list of commands, so it'll have only rootpath in the url.
Example:  
![image](https://user-images.githubusercontent.com/87537688/131329905-f6830a82-900b-4f92-ad24-4e8bcbdc8afd.png)